hawkauthlib/utils.py

```diff
@@ -118,6 +118,27 @@ def parse_authz_header(request, *default):
         raise
 
 
+def get_signature(request, key, params, algorithm=None):
```

Comment:
This is an internal version where we're assuming that the provided parameters are valid.

Reply:
Did you run the tests? `ALGORITHMS` is a circular reference now